e2e/scenario/scenario.go

```diff
@@ -77,6 +77,10 @@ func (t *Template) scenarios() []*Scenario {
 		t.marinerv2AirGap(),
 		t.azurelinuxv2AirGap(),
 		t.marinerv2ARM64AirGap(),
+		t.ubuntu2204SystemdChronyDropin(),
```

Comment:
could we look at ordering the scenarios in list group by OSFamily ? to make it easier to see what is being tested on which OS ?

Reply:
Good idea. I'll include this reordering in the PR.